src/workerd/api/node/util.c++

```diff
@@ -234,6 +234,10 @@ jsg::JsValue UtilModule::getBuiltinModule(jsg::Lock& js, kj::String specifier) {
   return js.undefined();
 }
 
+jsg::JsObject UtilModule::getEnvObject(jsg::Lock& js) {
+  return js.getEnv(true);
```

Comment:
what about:
getEnvObject -> getProcessEnvObj
getEnv -> getProcessEnv

Reply:
If feel strongly then.
Workerd has env and process.env.
They are different and it causes confusion. 
Unless I'm mistaken, the references here is to process.env?